code/game/gamemodes/blob/blob_report.dm

```diff
@@ -2,17 +2,16 @@
 
 /datum/game_mode/blob/send_intercept(var/report = 1)
 	var/intercepttext = ""
-	var/interceptname = "Error"
+	var/interceptname = ""
 	switch(report)
 		if(0)
 			..()
 			return
 		if(1)
-			interceptname = "Biohazard Alert"
-			intercepttext += "<FONT size = 3><B>Nanotrasen Update</B>: Biohazard Alert.</FONT><HR>"
+			intercepttext += "<FONT size = 3><B>NanoTrasen Update</B>: Biohazard Alert.</FONT><HR>"
 			intercepttext += "Reports indicate the probable transfer of a biohazardous agent onto [station_name()] during the last crew deployment cycle.<BR>"
-			intercepttext += "Preliminary analysis of the organism classifies it as a level 7 biohazard. Its origin is unknown.<BR>"
-			intercepttext += "Nanotrasen has issued a directive 7-10 for [station_name()]. The station is to be considered quarantined.<BR>"
+			intercepttext += "Preliminary analysis of the organism classifies it as a level 5 biohazard. Its origin is unknown.<BR>"
+			intercepttext += "NanoTrasen has issued a directive 7-10 for [station_name()]. The station is to be considered quarantined.<BR>"

```

Comment:
The overwhelming majority of "Nanotrasen" in our code is just "Nanotrasen", not "NanoTrasen"